public/app/core/components/AppChrome/ExtensionSidebar/ExtensionSidebar.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2 } from '@grafana/ui';
+
+import { useExtensionSidebarContext } from './ExtensionSidebarProvider';
+
+export const EXTENSION_SIDEBAR_WIDTH = '300px';
```

Comment:
This is the same width as the Mega menu. I'd prefer it to be a bit wider for an extension, but curious to get other opinions here.